src/tests/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs

```diff
@@ -691,6 +725,21 @@ public int RunTest(string executable, string outputFile, string errorFile, strin
                                         TryPrintStackTraceFromCrashReport(dmpFile.FullName, outputWriter);
                                     }
                                 }
+                                else
+                                {
+                                    var dmpFilesInfo = crashDumpFolderInfo.GetFiles("*.dmp").OrderByDescending(f => f.CreationTime);
```

Comment:
should we match against a pattern to rule out other crashes?

Reply:
I don't think so as we have some tests that use custom entry-point modules (in particular in the COM space) that we don't want to miss. The time filter should be good enough to ensure we don't get other random dump files IMO.